weave-js/src/components/Panel2/PanelObject.tsx

```diff
@@ -243,8 +252,10 @@ export const PanelObject: React.FC<PanelObjectProps> = props => {
         width: '100%',
         height: '100%',
         overflowY: 'auto',
+        paddingLeft: level === 0 ? '8px' : '0px',
+        paddingBottom: level === 0 ? '4px' : '0px',
       }}>
-      <KeyValTable.Table>
+      <div style={{display: 'table', fontSize: '13px'}}>
```

Comment:
why is `KeyValTable.Table` removed here? We should prefer that

Reply:
`KeyValTable.Table` has a padding of 2px which messes up the alignment when objects have nested fields. It's also used elsewhere in the app so I didn't want to change the default padding of the element.